elixir/README.md

```diff
@@ -509,7 +509,15 @@ terraform apply -var image_tag=$(terraform output -raw image_tag)
 
 ### Deploying production
 
-Go to ["Deploy Production"](https://github.com/firezone/firezone/actions/workflows/deploy.yml) CI workflow and click "Run Workflow".
+Before deploying, check if the `main` branch has any breaking changes since the last deployment. You can do this by comparing the `main` branch with the last deployed commit, which you can find [here](https://github.com/firezone/firezone/deployments/gcp_production).
+
+Here is a one-liner to open the comparison in your browser:
+
+```bash
+open "https://github.com/firezone/firezone/compare/$(curl -L -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" "https://api.github.com/repos/firezone/firezone/actions/workflows/deploy.yml/runs?status=completed&per_page=1" | jq -r '.workflow_runs[0].head_commit.id')...main"
+```
+
+Then, go to ["Deploy Production"](https://github.com/firezone/firezone/actions/workflows/deploy.yml) CI workflow and click "Run Workflow".
```

Comment:
I'm guessing this set of instructions is for what to do if there are not any breaking changes.  Is there a different workflow if there are breaking changes?

Reply:
@bmanifold if there are breaking changes how to deploy them is a decision that should be made with the whole team, because various changes might require various steps to deploy them (what is deployed first, do we need to email customers, grace period, etc.)

Reply:
Makes sense.  I added a suggestion comment above for this case.

Reply:
Thanks